buildlib/dockers-arm/push-release-images.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -eEx
+
+# shellcheck disable=SC2086
+basedir=$(cd "$(dirname $0)" && pwd)
+
+registry=harbor.mellanox.com/ucx/aarch64
```

Comment:
Is it correct harbor url? Shouldn't be the RDMZ one?

Reply:
RDMZ Harbor is a mirror of the lab one. Can't push directly to it.